.cargo/config

```diff
@@ -1,5 +1,14 @@
 [target.x86_64-pc-windows-msvc]
 rustflags = ["-C", "target-feature=+crt-static"]
 
+[target.'cfg(all(windows, debug_assertions))']
+rustflags = [
+  "-C",
+  "target-feature=+crt-static",
+  "-C",
+  # double the stack size to prevent swc overflowing the stack in debug
+  "link-arg=/STACK:2097152",
```

Comment:
Again...? 😬

Reply:
That happened in the past? FWIW swc’s tests on its repo requires increasing the stack size.

Reply:
Yeah, I believe we once increased stack size on Windows, but then there were some improvements that rendered it unneeded

Reply:
I think probably something must have increased on our side that just tipped it over. It never used to stack overflow on Windows with this swc version.